README

```diff
@@ -197,10 +197,11 @@ General notes
 
 - Open MPI has taken some steps towards Reproducible Builds
   (https://reproducible-builds.org/).  Specifically, Open MPI's
-  "configure" and "make" process, by default, records some
-  system-specific information such as the hostname where Open MPI was
-  built and the username who built it.  If you desire a Reproducible
-  Build, set the $USER and $HOSTNAME environment variables before
+  "configure" and "make" process, by default, records the build date
+  and some system-specific information such as the hostname where
+  Open MPI was built and the username who built it.
+  If you desire a Reproducible Build, set the $SOURCE_DATE_EPOCH,
```

Comment:
Should we put something here about the expected format of `$SOURCE_DATE_EPOCH` (e.g., a simple/trivial example)?

Reply:
Actually, it looks like the commit message has a URL that points to how to set it... but that URL is now stale (i.e., there's a new URL).

Maybe it's sufficient to a) update the commit message and b) also put that URL here in the README text.

Reply:
I updated README and the commit message.  Please review.

Reply:
README LGTM